### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ikocherovets**
- [MET landing](https://www.figma.com/file/lSR1m42L9YwzQwzzxKwHpw/THE-MET)
- [DEMO LINK](https://ikocherovets.github.io/layout_landing-page/)
this is not the latest version, lots of corrects will be done.

### CODE REVIEW

**Review by roman-mirzoian — CHANGES_REQUESTED** [review]
Good start, see work in progress. 
There is a two moments that you can fix with alignment:

and you also need to fix the page when the menu is open (when the menu is open, you can scroll below with the mouse wheel, the effect of which you see on the gif)

https://gyazo.com/9e999f8aace82efcd200827565dc5f26

**Comment by ikocherovets** [comment]
here is almost done version of my landing page, I will come back with that after some time, need to focus on JS Advanced (Alisa told us to do that)

**Review by roman-mirzoian — CHANGES_REQUESTED** [review]
I didn't see any changes in the demo, maybe you accidentally forgot to update deployment.

**Comment by ikocherovets** [comment]
@roman-mirzoian take my apologies, I totally forgot about that part of actions. Here is updated demo link to my landing page after `npm run deploy` command.

**Review by roman-mirzoian — CHANGES_REQUESTED** [review]
Good work, I can see progress in completing the task. But there is still work to be done.

Let's go through the [checklist](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md).

1) Use the logo as a favicon.
2) Use the landing name as a page title
example: `Logo` THE MET

image

3) All logos should be links to the top of the page
(these pictures do not have a link to the top of the page)
image

it is also better to add a small hover size for all pictures.

4) phone numbers, emails and links to maps must be clickable with a hover effect
image

image
https://gyazo.com/d67b011bba2e85470fc46d390bf9cc91

image
https://gyazo.com/5824b28dffe686283dd7fb01741ccc09

although you need to add an image below
image

**Comment by ikocherovets** [comment]
Hello dear mentors :)
Here is my landing page, I need to make the last corrects, specifically convert magic numbers to variables. In order to not waste time, I push my landing code for your review :)
[DEMO LINK](https://ikocherovets.github.io/layout_landing-page/)

File: src/blocks/reasons.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    font-size: 24px;
+    line-height: 34px;
+    text-align: center;
+    color: #d12d35;
+  }
+
+  &__category {
+    margin: 0;
+    font-family: Roboto, sans-serif;
+    font-weight: 400;
```
Comment (roman-mirzoian) [inline src/blocks/reasons.scss:23]:
```suggestion
```

File: src/blocks/section-title.scss [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      font-size: 48px;
+    }
+
+    &--align-left {
+      text-align: left;
+    }
+  }
+
+  &-description {
+    font-weight: 400;
```
Comment (roman-mirzoian) [inline src/blocks/section-title.scss:23]:
```suggestion
```

**Review by roman-mirzoian — CHANGES_REQUESTED** [review]
Good job, I see you have corrected most of my comments, there is still a bit to correct.

1) It is better to add a small hover size effect to these pictures.
https://gyazo.com/656b5c26d5df887cc6bc1f334d6f2921

2) Also necessary to add a hover to the links of email numbers and geolocations and to the buttons:
image
  https://gyazo.com/b784c6e2c630894cd3ad3b8c8f27522c

image

3) At a width of 320 pixels, the display, background of the header and footer is broken
  https://gyazo.com/5914bd65d6ae2fd7a6ea97e007ffe76b

**Review by roman-mirzoian — CHANGES_REQUESTED** [review]
Well done, but pay attention to these comments:
1) Here you need to change the color, not the size
https://gyazo.com/a1199039fb955809463b05db565b22ee
image

2) This guidance looks sharp and uneven, it is better to increase the size, for example, by 5 - 10% with a smooth transition
https://gyazo.com/4785b249d2610453ff12c3aa574612f6

3) There is also a small bug with the footer that can be easily fixed
https://gyazo.com/388d984180eb5825ea6b82e6b5697815

**Comment by ikocherovets** [comment]
Hi, Roman :) Thanks for the patience with all these comments which you've been giving to me. I know it can be annoying, but it's only because of them, I'm getting better.

**Review by roman-mirzoian — APPROVED** [review]
Well done, I see the correct edits in the code. I also see that you updated the deploy a few minutes ago, but for some reason it was not updated.

https://gyazo.com/ab90a2464298bbc2ef0718d84e48efa6

Don't worry and keep up the good work, you're doing great.

**Pull request closed without merging**